models/users.models.js

```diff
@@ -1,14 +1,20 @@
 const db = require("../db/connection");
 
 exports.checkUserExists = (username) => {
-    return db
-      .query("SELECT * FROM users WHERE username = $1", [username])
-      .then(({ rows }) => {
-        if (!rows.length) {
-          return Promise.reject({
-            status: 404,
-            msg: "Not Found: User does not exist",
-          });
-        }
-      });
-  };
\ No newline at end of file
+  return db
+    .query("SELECT * FROM users WHERE username = $1", [username])
```

Comment:
Not sure which endpoint this was for, but bear in mind that author/username is a foreign key so should fail via psql errors - just a little hint